wpeverest/user-registration
# UR-503 Tweak - New addons in extensions page

### All Submissions:






### Changes proposed in this Pull Request:
This PR adds two new add-ons ActiveCampaign and Two Factor Authentication to the list of addons in the extension page.

### How to test the changes in this Pull Request:

1. Go to the User Registration -> Extension page and click on the Refresh Templates button.
2. Check if two addons are added in the list.

### Types of changes:




### Other information:




### Changelog entry

> Tweak - New addons in extensions page
